Sources/MapboxDirections/RouteRefreshing.swift

```diff
@@ -0,0 +1,37 @@
+import Foundation
+
+/**
+ A skeletal route containing infromation to refresh `Route` object annotations.
+ */
+public protocol RouteRefreshing {
```

Comment:
If we want `Route` and `RefreshedRoute` to conform, could it be `RefreshableRoute`?

Reply:
Conformance to this protocol means that this object "may be used to refresh a route", but not "a route that can be refreshed". Naming is hard, but it's the best idea I had so far...

Reply:
What about `RouteRefreshSource`? Agree, difficult naming

Reply:
I like `RouteRefreshSource`. Updated the code